tests/unit/services/ajax-test.js

```diff
@@ -134,6 +134,49 @@ test("request() promise label is correct", function(assert) {
   assert.equal(postPromise._label, 'ember-ajax: POST to /posts');
 });
 
+test("post() promise label is correct", function(assert) {
+  service = Service.create();
+  const url = '/posts';
```

Comment:
Ok. I don't know that I'm entirely convinced of the usefulness of that distinction, but it's not an unreasonable explanation. Thanks for clarifying, @stefanpenner!